.github/workflows/cypress-testing.yml

```diff
@@ -36,11 +36,20 @@ jobs:
       - name: Start Anvil
         run: yarn test:anvil &
 
+      - name: Wait for Anvil
+        run: |
+          while ! curl -s http://localhost:8545; do
```

Comment:
Wouldn't this potentially be an infinite loop if for some reason it is never recheable?

Reply:
indeed it would. I was forgot the action would run for 6hrs waiting